### DISCUSSION TITLE
Adding Python Type stubs

### DISCUSSION THREAD

**Discussion by sbdchd** [bigcat88/pillow_heif#65] [Q&A] [answered]
Would you be open to me adding a PR to add type stubs?

If not, would you be okay with adding some stubs to the typeshed?

PS: thank you for the library!!

**Comment by bigcat88** [comment, marked as answer]
Yeah, feel free to make a Pull Request

**Comment by thcrt** [comment]
Hey! Sorry for necro-ing.
Did you ever end up writing these stubs? Obviously, no pressure to do so if you haven't already, I just thought I'd ask to avoid reinventing the wheel myself :^)

**Reply by sbdchd** [reply]
Hmm I don't think so 😬
